trade-finance-manager-ui/server/helpers/premium-payments-table-checkbox-id-helper.test.ts

```diff
@@ -8,29 +9,43 @@ import {
 import { PremiumPaymentsTableCheckboxId } from '../types/premium-payments-table-checkbox-id';
 
 describe('premium payments table checkbox id helper', () => {
-  describe('getFeeRecordIdsFromPremiumPaymentsCheckboxIds', () => {
-    it('extracts all the fee record ids in a single checkbox id', () => {
+  describe('getFeeRecordIdsFromPremiumPaymentsCheckboxId', () => {
+    it('extracts a single fee record id from a checkbox id containing a single id', () => {
+      // Arrange
+      const idList = [27];
+      const checkboxId: PremiumPaymentsTableCheckboxId = 'feeRecordIds-27-reportedPaymentsCurrency-GBP-status-TO_DO';
+
+      // Act
+      const extractedIds = getFeeRecordIdsFromPremiumPaymentsCheckboxId(checkboxId);
+
+      // Assert
+      expect(extractedIds).toEqual(idList);
+    });
+
+    it('extracts all the fee record ids from a checkbox id containing multiple ids', () => {
       // Arrange
       const idList = [1, 27, 314];
       const checkboxId: PremiumPaymentsTableCheckboxId = 'feeRecordIds-1,27,314-reportedPaymentsCurrency-GBP-status-TO_DO';
 
       // Act
-      const extractedIds = getFeeRecordIdsFromPremiumPaymentsCheckboxIds([checkboxId]);
+      const extractedIds = getFeeRecordIdsFromPremiumPaymentsCheckboxId(checkboxId);
```

Comment:
const has `Ids`, the function has `CheckboxId`, previously had `CheckboxIds`?

Reply:
This has been resolved now as the `Id` function can be merged with the `Ids` function